src/createAnimatedComponent.js

```diff
@@ -229,14 +229,25 @@ export default function createAnimatedComponent(Component) {
         : [this.props.style];
       styles = flattenArray(styles);
       const viewTag = findNodeHandle(this);
+      const viewName = this._obtainViewName();
       styles.forEach((style) => {
-        if (style && style.viewTag !== undefined) {
-          style.viewTag.value = viewTag;
+        if (style) {
+          if (style.viewTag) {
```

Comment:
This also can be flattened to `if(style?.viewTag)`

Reply:
fixed d1fa5c479790adcddb9622b1fce6edde61cee462